pkg/archive/archive.go

```diff
@@ -526,10 +530,11 @@ loop:
 
 		path := filepath.Join(dest, hdr.Name)
 		rel, err := filepath.Rel(dest, path)
+
```

Comment:
Perhaps this should stay as it was before because the `if err` check is relevant for the line which was right before it.